### PR TITLE
chore: added await in order to resolve the pending promise

### PR DESCRIPTION
**Pull request by anikethsaha**
**What kind of change does this PR introduce?**
bugfix/refactoring

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
NA

**Summary**
Added `await` in order to resolve the pending promise which was resulting in prompting error for 3rd question in init command

**Does this PR introduce a breaking change?**
NA

**Other information**
NA

### CODE REVIEW

File: packages/generators/init-generator.ts [hunk 1]
```diff
@@ -133,6 +133,8 @@ export default class InitGenerator extends Generator {
 				chunkFilename: "'[name].[chunkhash].js'",
 				filename: "'[name].[chunkhash].js'"
 			};
+		} else {
```
Comment (evenstensberg) [inline packages/generators/init-generator.ts]: remove this

Comment (anikethsaha) [reply packages/generators/init-generator.ts]: but what about the output directory then ?....if it is specified by the user then shouldn't it be added ?

Comment (evenstensberg) [reply packages/generators/init-generator.ts]: If not specified we are setting a default one in webpack-cli no need to add it to config

Comment (anikethsaha) [reply packages/generators/init-generator.ts]: What about when its specified?

Comment (evenstensberg) [reply packages/generators/init-generator.ts]: I think we're setting that prop further up the Lines.

Comment (anikethsaha) [reply packages/generators/init-generator.ts]: I guess it was not adding any, ok will test this particular one more time and will get back here

Comment (anikethsaha) [reply packages/generators/init-generator.ts]: @evenstensberg I checked with when specified the output directory, `output.path` not reflecting in the config file, its simply giving the chunk file support.

Comment (evenstensberg) [reply packages/generators/init-generator.ts]: I'm working on these defaults next so leave them to me, need to check in with some folks and speak to various of people to set the correct defaults

Comment (anikethsaha) [reply packages/generators/init-generator.ts]: sure then!!

**Comment by anikethsaha** [comment]
yeah, messed up the rebasing. trying to fix it. 👍

File: packages/generators/init-generator.ts (hunk 1 above)
Comment (anikethsaha) [reply packages/generators/init-generator.ts]: done

**Pull request merged**